net/matrix-structs/Portfile

```diff
@@ -3,6 +3,7 @@
 PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           cmake 1.1
+PortGroup           obsolete 1.0
```

Comment:
If using the `obsolete` portgroup, much of what's in the portfile should be removed so that it is closer to a stub port, e.g. nothing should be fetched when attempting to install it. The `revision` needs to be increased, and it would be a good idea to add a comment indicating that the stub port can be removed in one year:

```
PortSystem          1.0
PortGroup           obsolete 1.0

# Remove after 2021-09-28
name                matrix-structs
version             2018-07-14
revision            1
categories          net devel
replaced_by         mtxclient
```

Reply:
Just to confirm, you want me to remove the github.setup and other stuff too?

Reply:
Yes, `github.setup` would no longer be needed. See https://guide.macports.org/chunked/development.practices.html#development.obsolete-portgroup for explanation/example.

Reply:
Thanks, I've made it a stub now